src/local-history/local-history-manager.ts

```diff
@@ -242,7 +243,20 @@ export class LocalHistoryManager {
     }
 
     /**
-     * Remove the active revision of a file.
+     * Recursively deletes all empty folders in the workspace revision directory.
+     */
+    private removeEmptyFoldersRec(): void {
+        if (!vscode.workspace.workspaceFolders || vscode.workspace.workspaceFolders.length <= 0) {
+            return;
+        }
+
+        const workspacePath = vscode.workspace.workspaceFolders![0].uri;
```

Comment:
@kaiyue0329 will this fail in a multiple-root workspace? (this gets the first workspace root only).

Reply:
@vince-fugnitto yes, I will update the code.

Reply:
I'm thinking you can use `getWorkspaceFolder` to give you the workspace name for the given `uri` and you use that to construct the path?:

<div align='center'>

![Screenshot from 2020-06-08 11-52-08](https://user-images.githubusercontent.com/40359487/84052463-9a6c8c80-a97e-11ea-88a1-fcd04da69e02.png)


</div>